src/transformers/models/clip/tokenization_clip.py

```diff
@@ -315,7 +315,7 @@ def __init__(
 
             self.fix_text = ftfy.fix_text
         except ImportError:
-            logger.warning("ftfy or spacy is not installed using BERT BasicTokenizer instead of ftfy.")
+            logger.info("ftfy or spacy is not installed using custom BasicTokenizer instead of ftfy.")
```

Comment:
Is it really necessary to throw a warning here? I think just a logger info is good enough no? 
Also using "BERT" is a bit confusing here IMO.